core/arch/arm/arm.mk

```diff
@@ -18,3 +18,10 @@ core-platform-cppflags += $(arm32-platform-cppflags)
 core-platform-cflags += $(arm32-platform-cflags)
 core-platform-aflags += $(arm32-platform-aflags)
 endif
+
+ifeq ($(CFG_WITH_PAGER),y)
+ifneq ($(CFG_DISABLE_CONCURRENT_EXEC),y)
```

Comment:
case 1) (without message) and 2) are fine with me.

In case 3), I would raise an error anyway, as it is now. Nobody looks at warnings. We may change the name from `force` to `check_or_set` (well, I'm not very good at naming).

Can the reason be optional?